nginx-odoo.py

```diff
@@ -37,7 +37,7 @@ def get(self):
                 if not redirect_url.endswith("/"):
                     redirect_url += "/"
                 return self.redirect(f"{redirect_url}auth/{session_id}")
-        self.render(r"./templates/login.html", **config.theme_params)
+        self.render(r"./templates/login-2.html", **config.theme_params)
```

Comment:
@KKamaa We currently have an environment variable for the backgoundcolor; could you create another environment variable called `NGINX_ODOO_THEME` or something, which defaults to `login`, but with which you can configure to use another theme?

Reply:
@thomaspaulb added the env variable